aicsimageio/readers/czi_reader.py

```diff
@@ -217,3 +217,16 @@ def _is_multiscene(self):
     def get_channel_names(self, scene: int = 0):
         chelem = self.metadata.findall("./Metadata/Information/Image/Dimensions/Channels/Channel")
         return [ch.get("Name") for ch in chelem]
+
+    # TODO refactor this utility function into a metadata wrapper class
+    def _getmetadataxmltext(self, findpath, default=None):
+        ref = self.metadata.find(findpath)
+        if ref is None:
+            return default
+        return ref.text
+
+    def get_physical_pixel_size(self, scene: int = 0):
```

Comment:
Two nitpicky things: I personally think this should be a tuple over a list and we should have a return type hint.

```suggestion
    def get_physical_pixel_size(self, scene: int = 0) -> Tuple(int):
```

Reply:
Or if we wanted to be fancy lads, a `NamedTuple` so that people can say `im.get_physical_pixel_size().x`